src/views/Datasets.vue

```diff
@@ -431,17 +428,24 @@ const fields = [
 	},
 ]
 
-function getApiError(error) {
-	let apiError = "API Error"
+function getApiError(error,apiCall,datasetId) {
+	let errorText = "Error "
+	errorText+=apiCall
+	errorText+=datasetId
```

Comment:
should it be something like:
let errorText = ["Error", apiCall, datasetId].join(" ")

Reply:
use similar kind of things below, having whitespace characters in the strings looks nasty

Reply:
This is a good suggestion. applied!

Reply:
changed code